src/js/dqa-scores.jsx

```diff
@@ -0,0 +1,357 @@
+var React = require('react'),
+    _ = require('underscore');
+
+var FixedDataTable = require('fixed-data-table'),
+    Column = FixedDataTable.Column,
+    Table = FixedDataTable.Table;
+
+var isColumnResizing;
+
+var SortTypes = {
+    ASC: 'ASC',
+    DESC: 'DESC',
+};
+
+var Status = {
+    'new': {
+        sortOrder: 0,
+        desc: 'The issue has been identified by the Data Coordinating Center.',
+    },
+    'under review': {
+        sortOrder: 1,
+        desc: 'The site has acknowledged that it needs to investigate the issue.',
+    },
+    'solution proposed': {
+        sortOrder: 2,
+        desc: ('The site has investigated the issue, and ' +
+               'a consensus about the solution has been reached; ' +
+               'the site has agreed to fix the issue in the next round.'),
+    },
+    'default': {
+        sortOrder: 10,
+    },
+    'persistent': {
+        sortOrder: 20,
+        label: 'inherent',
+        desc: ('The issue has been investigated, ' +
+               'and it was concluded that it cannot be fixed'),
+    },
+    'withdrawn': {
+        sortOrder: 30,
+        desc: 'False alarm.  An investigation has concluded this to be a non-issue.',
+    },
+    'non-issue': {
+        sortOrder: 31,
+        desc: 'False alarm.  An investigation has concluded this to be a non-issue.',
+    },
+    'closed': {
+        sortOrder: 32,
+        desc: '',
+    },
+};
+        
+var DQAScores = React.createClass({
+    propTypes: {
+        columnWidth: React.PropTypes.number,
+        data: React.PropTypes.object,
+    },
+
+    getDefaultProps: function() {
+        return {
+            columnWidth: 120,
+            data: {},
+        };
+    },
+
+    getInitialState: function() {
+        return {
+            columnWidths: {},
+            isCollapsed: false,
+            rows: {},
+            sortBy: null,
+            sortDir: null,
+            statusList: [],
+            tableWidth: 500,
+        };
+    },
+
+    _initRows: function(props) {
+        var i;
+        var row;
+        var rows = [];
+        var score;
+        var site;
+        
+        var firstSiteData = props.data[Object.keys(props.data)[0]];
```

Comment:
All sites will have the same set of keys, so any site will do (The same set of issues is reported for each site, and the number of issues is set to 0 if the site has no issues like that).
I'll rename the variable -- you are right, it shouldn't be named 'first' if it's any site.